BucketeerTests/EvaluationForegroundTaskTests.swift

```diff
@@ -93,7 +93,7 @@ final class EvaluationForegroundTaskTests: XCTestCase {
         let task = EvaluationForegroundTask(
             component: component,
             queue: dispatchQueue,
-            retryPollingInterval: 1,
+            retryPollingInterval: 1000,
```

Comment:
1 millisecond is too low and the test could fail sometimes, so I'm increasing it by 1 second.